sami2py/__init__.py

```diff
@@ -20,7 +20,10 @@
 import sys
 import os
 
-__version__ = str('0.2.0')
+# set the version
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, 'version.txt')) as version_file:
+    __version__ = version_file.read().strip()
 
 # get home directory
 home_dir = os.path.expanduser('~')
```

Comment:
Recommend deleting variables you don't want users to have access to at the top level of the module

Reply:
cleaned.  Added #151 as a breaking change to further simplify storage of required paths